packages/optimizer/lib/transformers/AutoExtensionImporter.js

```diff
@@ -141,7 +141,23 @@ class AutoExtensionImporter {
     if (!this.componentVersions) {
       this.componentVersions = {};
       for (const component of params.componentVersions) {
-        this.componentVersions[component.name] = component.latestVersion;
+        if (Array.isArray(component.version)) {
```

Comment:
Is this meant to replicate the `latestVersion` behavior? If yes, it is incorrect AFAICT. At some point in the past latestVersion stopped correlating with the highest number